pytest.ini

```diff
@@ -3,7 +3,7 @@ env =
     FLASK_ENV=unit_test
     FLASK_DEBUG=1
     FUND_STORE_API_HOST=https://funding-service-design-fund-store-test.london.cloudapps.digital
-    D:DATABASE_URL=postgresql://postgres:postgres@127.0.0.1:5432/fsd_assess_store_test
+    D:DATABASE_URL=postgresql://postgres:postgres@127.0.0.1:5432/fsd_assess_store
```

Comment:
When is this used? Shouldn't it match one of the database names defined in the config?

Reply:
Good point! I think this must be an artefact of before we had the config management we have now. Will remove!